platforms/ide/tooling-api/build.gradle.kts

```diff
@@ -133,10 +133,12 @@ tasks.named("toolingApiShadedJar") {
     dependsOn(gradle.includedBuild("build-logic").task(":java-api-extractor:assemble"))
 }
 
-integTest.usesJavadocCodeSnippets = true
 testFilesCleanup.reportOnly = true
 
 apply(from = "buildship.gradle")
 tasks.isolatedProjectsIntegTest {
     enabled = false
 }
+
+// AutoTestedSamplesToolingApiTest including customized test logic
```

Comment:
```suggestion
// AutoTestedSamplesToolingApiTest includes customized test logic, so automatic auto testing samples generation is not needed (and would fail) in this project
```